libpod/util.go

```diff
@@ -0,0 +1,34 @@
+package libpod
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// WriteFile writes a provided string to a provided path
+func WriteFile(content string, path string) error {
```

Comment:
Does this need to be in libpod? I feel like this should be in cmd/kpod/common.go

Reply:
Leaving for now